cluster/examples/openshift-ovs-cni.yml

```diff
@@ -26,31 +26,6 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
-      initContainers:
```

Comment:
I think we don't need separate manifest for OpenShift and for Kubernetes. Could you please check whether they are the same now?